app/services/spotify_service.rb

```diff
@@ -29,26 +29,34 @@ def self.authenticate(code)
   end
 
   def self.get_recent_albums_for(user:, released_after:)
```

Comment:
Lets rename this variable to be more descriptive. released_after sounds like a boolean